website/docs/r/data_factory_linked_service_azure_blob_storage.html.markdown

```diff
@@ -44,7 +44,11 @@ resource "azurerm_data_factory_linked_service_azure_blob_storage" "example" {
 
 The following supported arguments are common across all Azure Data Factory Linked Services:
 
-* `name` - (Required) Specifies the name of the Data Factory Linked Service. Changing this forces a new resource to be created. Must be globally unique. See the [Microsoft documentation](https://docs.microsoft.com/en-us/azure/data-factory/naming-rules) for all restrictions.
+* `name` - (Required) Specifies the name of the Data Factory Linked Service.
+  Changing this forces a new resource to be created. Must be unique within a data
+  factory. See the [Microsoft
+  documentation](https://docs.microsoft.com/en-us/azure/data-factory/naming-rules)
+  for all restrictions.
```

Comment:
```suggestion
* `name` - (Required) Specifies the name of the Data Factory Linked Service. Changing this forces a new resource to be created. Must be unique within a data factory. See the [Microsoft documentation](https://docs.microsoft.com/en-us/azure/data-factory/naming-rules) for all restrictions.
```

This got broken up over multiple lines somehow.

Reply:
Probably one of my magic plugins on VS Code 😋